qaframework-bdd-tests/src/test/resources/features/refapp-2.x/15-RolesAndPrivileges/AddUserAndAddRoleToUser.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Add User and Add Roles To User
+
+  Background: 
+    Given a user clicks on system administration app
+    Then the system loads manage users page
+
+  @selenium 
+  @addUserAndAddRolesToUser
+  Scenario: Adding role to user
+    When a user fills in person details
+    Then a user assigns roles to the created user
+
+  @selenium 
+  @addUserAndAddRolesToUser
+  Scenario: Adding user
+    When a user clicks the add user link
+    Then the system loads the create new user page
+    And a user enters the details of the user
+    And a user logins into the system as the created user
+    And a user logins into the system as an admin
+    And a user deletes the user
```

Comment:
take a close look at the gherkin step!

Reply:
i dont think i get your review here @kdaud , kindly elaborate!

Reply:
Take a close look at the gherkin step
```js
And a user deletes the user
```

Reply:
is there something wrong with it? you can point it out please!

Reply:
what is being deleted? is it the user or user account?

Reply:
yes it is

Reply:
then worth adding the word account.

```js
And a user deletes the user account
```

Reply:
resolved!!